Week4/Blockbuster/app/src/main/java/com/example/blockbuster/fragments/AddMovieFragment.kt

```diff
@@ -0,0 +1,95 @@
+package com.example.blockbuster.fragments
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.navigation.findNavController
+import com.example.blockbuster.data.DataManager
+import com.example.blockbuster.data.Movie
+import com.example.blockbuster.R
+import kotlinx.android.synthetic.main.fragment_add_movie.*
+
+class AddMovieFragment : Fragment() {
```

Comment:
You went above and beyond using fragments and adding this add functionality to the app, very impressive.